templates/workspace_services/guacamole/user_resources/guacamole-azure-windowsvm/porter.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 schemaVersion: 1.0.0
 name: tre-service-guacamole-windowsvm
-version: 0.7.9
+version: 0.7.10
```

Comment:
If we do an upgrade of a VM with this bundle what does it do?

If we are doing a image change with the same name we might want to a major version in case to prevent upgrade?

Reply:
I have managed to successfully upgrade a Win11 VM from 0.7.9 to 0.7.10. 

I'm happy to update this to a major version change as a precaution if you still think it necessary

Reply:
Hmm, did it not create a new OS disk, did the data in the VM remain after the upgrade?

Reply:
Yes it did, so data does get destroyed. I will push a change to a major version